identity-iota/src/error.rs

```diff
@@ -57,4 +57,8 @@ pub enum Error {
   NoExplorerURLSet,
   #[error("Invalid Explorer Url")]
   InvalidExplorerURL,
+  #[error("compression Error")]
+  CompressionError,
+  #[error("Invalid Message Flags")]
```

Comment:
Nit: Error messages should be lower case (yes I know we are currently doing this wrong). Hence it should be `#[error("invalid message flags")]`, same goes for `#[error("compression error")]` above.